site/profile/manifests/cvmfs.pp

```diff
@@ -90,6 +100,16 @@
     }
   }
 
+  consul_template::watch { '/etc/cvmfs/config.d':
```

Comment:
Since there is nothing in `config.d.epp` that requires consul, it would be easier to just make it a puppet template and remove the consul watch.

Reply:
Done